test/reducers/columnStateHelper-test.js

```diff
@@ -2,6 +2,7 @@
  * Copyright Schrodinger, LLC
  */
 
+import PrefixIntervalTree from 'PrefixIntervalTree';
```

Comment:
changes for tests incoming

most of them are because:
* `rows` renamed to `rowsToRender` 
* `columnAnchor` had failed some asserts